### PR TITLE
added new projects

### PR DESCRIPTION
**Pull request by scottgallant**
### General Contributing:

* [ ] Have you followed the guidelines in our [Contributing document](https://github.com/tinacms/tinacms.org/blob/master/CONTRIBUTING.md)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### All New Content Submissions: (To be confirmed by reviewer)

* [ ] Title is short & specific
* [ ] Headers are logically ordered & consistent
* [ ] Purpose of document is explained in the first paragraph
* [ ] Procedures are tested and work
* [ ] Any technical concepts are explained or linked to
* [ ] Document follows structure from templates
* [ ] All links work
* [ ] Spelling and grammer checker has been run
* [ ] Graphics and images are clear and useful
* [ ] Any prerequisites and next steps are defined.

### CODE REVIEW

**Comment by scottgallant** [comment]
I want to run this by Tyler from NASA before merging to ensure he's good with this.

File: content/blocksPages/showcase.json [hunk 1]
```diff
@@ -20,6 +20,15 @@
           },
           "_template": "project"
         },
+        {
+          "headline": "Community Coodinated Modeling Center",
```
Comment (jamespohalloran) [inline content/blocksPages/showcase.json:24]:
```suggestion
          "headline": "Community Coordinated Modeling Center",
```

**Review by jamespohalloran — DISMISSED** [review]

File: content/blocksPages/showcase.json (hunk 1 above)
Comment (scottgallant) [reply content/blocksPages/showcase.json]: Thank you, sir!

**Comment by scottgallant** [comment]
Approved! We're good to merge this.

**Review by perkinsjr — APPROVED** [review]

**Pull request merged**